src/lib/Scenes/Fair/Components/FairFollowedArtistsRail.tsx

```diff
@@ -1,97 +1,50 @@
-import { ActionType, ContextModule, OwnerType, TappedArtworkGroup } from "@artsy/cohesion"
+import { ActionType, ContextModule, OwnerType } from "@artsy/cohesion"
 import { FairFollowedArtistsRail_fair } from "__generated__/FairFollowedArtistsRail_fair.graphql"
-import { ArtworkTileRailCardFragmentContainer as ArtworkTileRailCard } from "lib/Components/ArtworkTileRail"
+import { SmallArtworkRail } from "lib/Components/ArtworkRail/SmallArtworkRail"
+import { SectionTitle } from "lib/Components/SectionTitle"
 import { navigate } from "lib/navigation/navigate"
-import { compact } from "lodash"
-import { Box, Spacer, Text } from "palette"
+import { extractNodes } from "lib/utils/extractNodes"
+import { Flex } from "palette"
 import React from "react"
-import { FlatList, TouchableOpacity } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 import { useTracking } from "react-tracking"
 
 interface FairFollowedArtistsRailProps {
   fair: FairFollowedArtistsRail_fair
 }
 
-export const FairFollowedArtistsRail: React.FC<FairFollowedArtistsRailProps> = ({
-  fair,
-  ...rest
-}) => {
-  if (!fair.followedArtistArtworks?.edges?.length) {
+export const FairFollowedArtistsRail: React.FC<FairFollowedArtistsRailProps> = ({ fair }) => {
+  if (!fair.filterArtworksConnection?.edges?.length) {
```

Comment:
Can we move this below line 21 and change it to

```suggestion
  if (!artworks.length) {
```